src/components/OrganizationsList/index.jsx

```diff
@@ -1,159 +1,171 @@
 import React, {Component} from 'react';
-import { Table, Button, InputGroup, InputGroupAddon, Input } from 'reactstrap';
 import Api from '../../utils/api';
-import Item from './Item';
 import AlertMessage from '../../sharedComponents/AlertMessage';
-import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+import ReactTable from 'react-table';
+import 'react-table/react-table.css';
+import './OrganizationsList.css'
 
 export default class DriversList extends Component {
 
-  state = {
-    organizations: [],
-    errors: null,
-    flash: null,
-    selectedPage: 1,
-    name: ''
+  constructor() {
+    super()
+    this.state = {
+      taxiSitesRaw: [],
+      taxiSitesList: [],
+      loading: true,
+      currentPage: 0,
+      pages: 0,
+      errors: null,
+      name: '',
+      flash: null,
+      siteName: ''
+    }
   }
 
   componentDidMount() {
-    const flash = this.props.location.flash;
-    if (flash) this.setState({flash});
-    Api.get('/organizations')
+    const flash = this.props.location.flash
+    if(flash) {
+      this.setState({flash})
+    }
+    this.fetchTaxiSites()
+  }
+
+  confirmDelete = (siteId) => {
+    var option = window.confirm("Eliminar?")
+    if (option === true) {
```

Comment:
Just leaving an if(boolean) can lead to truthy values. But yeah, maybe it's not needed there, because, yeah, I guess It's only true or false there. I will change that